stingray/lightcurve.py

```diff
@@ -451,8 +451,8 @@ def join(self, other):
             except IndexError:
                 count2 = None
 
-            if not count1 is None:
-                if not count2 is None:
+            if count1 is not None:
+                if count2 is not None:
```

Comment:
Ah, yes, that wasn't what I meant. I was actually wondering whether we can leave out the `is None`:
So instead of `if foo is not None:` write `if not foo:`.

Experimentation tells me that works, too, but might not be what we want here?